samples/nrf_desktop/configuration/app_nrf52840_pca20041.conf

```diff
@@ -64,6 +64,8 @@ CONFIG_CLOCK_CONTROL_NRF5_K32SRC_BLOCKING=y
 CONFIG_BT_DEVICE_NAME="Mouse nRF52 Desktop"
 CONFIG_BT_DEVICE_APPEARANCE=962
 
+CONFIG_BT_CTLR_CONN_PARAM_REQ=n
```

Comment:
What's the difference when your remove this option? Will mouse stop sending the preferred conn params requirest?

Reply:
we don't need it as mouse is sending that anyway from the app
I think I removed it to avoid doing it twice